DatadogSDK.podspec

```diff
@@ -19,6 +19,7 @@ Pod::Spec.new do |s|
   s.source = { :git => "https://github.com/DataDog/dd-sdk-ios.git", :tag => s.version.to_s }
   
   s.source_files = ["Sources/Datadog/**/*.swift",
+                    "Sources/Bridge/**/*.swift",
                     "Sources/_Datadog_Private/**/*.{h,m}",
```

Comment:
If the bridge integrates with RN SDK through `podspec`, I'd rather create a separate podspec: `DatadogBridge.podspec` and laid out the module in there. There's no point of including all the bridge files for Carthage and SPM distribution. It makes the binary and dSYMs files bigger, it adds bridge symbols into public namespace making the autocompletion harder / misleading and it impacts the code coverage for the SDK (although this depends on how we're going to test bridge's implementation).

I think this is sth that we can take care of, WDYT @buranmert ?

Reply:
yes, sounds good 👍 
we should also exclude `Bridge` folder the project to prevent Carthage artifact from having them